common/cosmwasm-smart-contracts/vesting-contract/Cargo.toml

```diff
@@ -6,8 +6,10 @@ edition = "2021"
 [dependencies]
 cosmwasm-std = "1.0.0"
 mixnet-contract-common = { path = "../mixnet-contract" }
+contracts-common = { path = "../contracts-common" }
 serde = { version = "1.0", features = ["derive"] }
 schemars = "0.8"
+log = "0.4"
```

Comment:
I assume it was only added for the purposes of debugging, but I don't think we should be adding that in prod code (especially since all stdout/stderr is completely ignored in a smart contract anyway)

Reply:
Fix in follow up PR